products/joomla.md

```diff
@@ -18,8 +18,8 @@ releases:
     eol: false
     support: true
     releaseDate: 2021-08-17
-    latest: "4.2.7"
-    latestReleaseDate: 2023-01-30
+    latest: "4.2.8"
```

Comment:
Not yet in 4.2 changelog, opened https://github.com/joomla/joomla-websites/issues/1868.